### PR TITLE
[BUGFIX] [MER-813] Persistent age check

### PR DESCRIPTION
**Pull request by mcass19**
https://eliterate.atlassian.net/browse/MER-813

### CODE REVIEW

File: priv/repo/migrations/20220214143811_update_users_tables_age_verified.exs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+defmodule Oli.Repo.Migrations.UpdateUsersTablesAgeVerified do
+  use Ecto.Migration
+  import Ecto.Query, warn: false
+
+  def change do
+    alter table(:users) do
+      add :age_verified, :boolean, default: true, null: false
```
Comment (mcass19) [inline priv/repo/migrations/20220214143811_update_users_tables_age_verified.exs]: We assume that created users will have this field in true, and if you don't use this check feature, the value will default to true as well. 

Is that ok for you?

Comment (jrissler) [reply priv/repo/migrations/20220214143811_update_users_tables_age_verified.exs]: I think we need to reverse that for potential legal reasons. If we haven't confirmed they are 13 - we cannot assume they are.

Comment (jrissler) [reply priv/repo/migrations/20220214143811_update_users_tables_age_verified.exs]: Also means you can avoid the query in migration.

**Review by jrissler — APPROVED** [review]
Approved after feedback is addressed.

File: lib/oli/accounts/schemas/user.ex [hunk 2]
```diff
@@ -191,6 +204,9 @@ defmodule Oli.Accounts.User do
         false
     end
   end
+
+  defp is_age_verification_enabled(_changeset),
```
Comment (gastonabella) [inline lib/oli/accounts/schemas/user.ex:208]: Do we need to pass the `changeset` in if we never use it?

**Review by gastonabella — APPROVED** [review]
LGTM ✅ Only a minor comment

File: lib/oli/accounts/schemas/user.ex (hunk 2 above)
Comment (mcass19) [reply lib/oli/accounts/schemas/user.ex:208]: We need since we are chaining this function in the validations as well

**Pull request closed without merging**